package/sbin/source_ports_validator.py

```diff
@@ -0,0 +1,42 @@
+import os
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def is_valid_port(raw_port: str) -> bool:
+    return raw_port.isdigit() and int(raw_port) < 10000
+
+
+def validate_source_ports(sources: list[str]) -> None:
+    source_ports = []
+    for source in sources:
+        tcp_ports = os.getenv(f"SC4S_LISTEN_{source}_TCP_PORT", "disabled").split(",")
+        udp_ports = os.getenv(f"SC4S_LISTEN_{source}_UDP_PORT", "disabled").split(",")
+        tls_ports = os.getenv(f"SC4S_LISTEN_{source}_TLS_PORT", "disabled").split(",")
+
+        source_ports.extend((source, port, "TCP") for port in tcp_ports)
+        source_ports.extend((source, port, "UDP") for port in udp_ports)
+        source_ports.extend((source, port, "TLS") for port in tls_ports)
+
+
+    busy_ports = set()
+    for source, port, proto in source_ports:
+        env_var = f"SC4S_LISTEN_{source}_{port}_PORT"
+
+        if port in ["disabled", ""]:
+            continue
+        elif not is_valid_port(port):
+            logger.error(f"{env_var}: {port} should be valid int [0, 10000]")
```

Comment:
message is like this:
```
SC4S_LISTEN_MERAKI_SWITCHES_514_PORT: Impossible to use default ports like 514
```

Can you make it more like this:
```
[source_ports_validator.py ERROR] Invalid source port detected for MERAKI_SWITCHES: 514/udp. Must be within the range [0, 1000].
[source_ports_validator.py ERROR] Invalid source port detected for MERAKI_SWITCHES: 514/udp. Default port usage not allowed (514, 614, 6514).
[source_ports_validator.py ERROR] Invalid source port detected for MERAKI_SWITCHES: 514/udp. Port number is not unique and has already been used for another source.
```

Reply:
I using `env var`, because it would be clear what you need to fix.
When you writing `MERAKI_SWITCHES` it would be unclear for users what fix

Reply:
but can we include the log level and logging utility name?

Reply:
discussed and decided not to include the log level and utility name